packages/resolver/lib/resolver.ts

```diff
@@ -3,17 +3,26 @@ const expect = require("@truffle/expect");
 const provision = require("@truffle/provisioner");
 
 import { ResolverSource } from "./source";
-import { constructSources } from "./sources";
+import { EthPMv1, NPM, GlobalNPM, FS, Truffle } from "./sources";
 
 export class Resolver {
   options: any;
   sources: ResolverSource[];
 
-  constructor(options: any) {
+  constructor(options: any, includeTruffleSource: boolean) {
```

Comment:
Is there a better way to denote that the resolver being instantiated should include the `Truffle` source? Do we want to eventually include the `Truffle` source in all resolvers?

Reply:
Eventually: definitely yes.
Now: maybe yes.

If the answer to "now" is "no", then we should do this:
```suggestion
  constructor(options: any, includeTruffleSource: boolean = false) {
```